tests/nat/endpointIndependentNAT.test.ts

```diff
@@ -1,372 +1,383 @@
-        },
-        dataDir,
-      ));
-      expect(exitCode).toBe(1);
-      expect(JSON.parse(stdout)).toEqual({
-        success: false,
-        message: 'No response received',
-      });
-      // But Agent 2 can ping Agent 1 now because it's expecting a response
-      ({ exitCode, stdout } = await testNatUtils.pkExecNs(
-        userPid!,
-        agent2Pid!,
-        ['nodes', 'ping', agent1NodeId, '--format', 'json'],
-        {
-          PK_NODE_PATH: agent2NodePath,
-          PK_PASSWORD: password,
-        },
-        dataDir,
-      ));
-      expect(exitCode).toBe(0);
-      expect(JSON.parse(stdout)).toEqual({
-        success: true,
-        message: 'Node is Active.',
-      });
-      // Can now ping Agent 2 (it will also be expecting a response)
-      ({ exitCode, stdout } = await testNatUtils.pkExecNs(
-        userPid!,
-        agent1Pid!,
-        ['nodes', 'ping', agent2NodeId, '--format', 'json'],
-        {
-          PK_NODE_PATH: agent1NodePath,
-          PK_PASSWORD: password,
-        },
-        dataDir,
-      ));
-      expect(exitCode).toBe(0);
-      expect(JSON.parse(stdout)).toEqual({
-        success: true,
-        message: 'Node is Active.',
-      });
-      await tearDownNAT();
-    },
-    global.defaultTimeout * 2,
-  );
-  test(
-    'Node1 behind EIM NAT cannot connect to Node2 behind EDM NAT',
-    async () => {
-      const {
-        userPid,
-        agent1Pid,
-        agent2Pid,
-        password,
-        dataDir,
-        agent1NodePath,
-        agent2NodePath,
-        agent1NodeId,
-        agent2NodeId,
-        tearDownNAT,
-      } = await testNatUtils.setupNATWithSeedNode('eim', 'edmSimple', logger);
-      // Since one of the nodes uses EDM NAT we cannot punch through
-      let exitCode, stdout;
-      ({ exitCode, stdout } = await testNatUtils.pkExecNs(
-        userPid!,
-        agent2Pid!,
-        ['nodes', 'ping', agent1NodeId, '--format', 'json', '-vv'],
-        {
-          PK_NODE_PATH: agent2NodePath,
-          PK_PASSWORD: password,
-        },
-        dataDir,
-      ));
-      expect(exitCode).toBe(1);
-      expect(JSON.parse(stdout)).toEqual({
-        success: false,
-        message: 'No response received',
-      });
-      ({ exitCode, stdout } = await testNatUtils.pkExecNs(
-        userPid!,
-        agent1Pid!,
-        ['nodes', 'ping', agent2NodeId, '--format', 'json', '-vv'],
-        {
-          PK_NODE_PATH: agent1NodePath,
-          PK_PASSWORD: password,
-        },
-        dataDir,
-      ));
-      expect(exitCode).toBe(1);
-      expect(JSON.parse(stdout)).toEqual({
-        success: false,
-        message: 'No response received',
+    });
+    afterEach(async () => {
+      await fs.promises.rm(dataDir, {
+        force: true,
+        recursive: true,
       });
-      await tearDownNAT();
-    },
-    global.defaultTimeout * 2,
-  );
-});
+    });
+    test(
+      'Node1 behind EIM NAT connects to Node2',
```

Comment:
Can you make your test descriptions lowercase, all of our test descriptions are lower case atm. And it should say `node 1 behind ...`.

Also what is EIM? I'm not familiar with this acronym, perhaps it should be expanded?